cargo-concordium/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+- Fix integration tests should panic if error occurs.
```

Comment:
This message is not very clear. Also the tool does not panic when tests fail now. It just  prints an error and returns a non-zero status code.